tests/csm/rest/test_query_deployment.py

```diff
@@ -42,14 +45,23 @@ def setup_class(cls):
         cls.ha_obj = HAK8s()
         cls.deploy_lc_obj = ProvDeployK8sCortxLib()
         cls.csm_obj = csm_api_factory("rest")
-        cls.csm_conf = configmanager.get_config_wrapper(fpath="config/csm/test_rest_capacity.yaml")
+        cls.csm_conf = configmanager.get_config_wrapper(
+            fpath="config/csm/test_rest_query_deployment.yaml")
         cls.update_seconds = cls.csm_conf["update_seconds"]
+        cls.failed_pod = None
+        cls.kvalue = None
+        cls.deploy_list = cls.csm_obj.master.get_deployment_name(POD_NAME_PREFIX)
+        cls.random_string = ''.join(random.choices(string.ascii_letters + string.digits, k=10))
+        cls.random_number = cls.csm_obj.random_gen.randrange(1, 99999)
+        cls.random_symbols = ''.join(random.choices(string.punctuation, k=10))
 
     def setup_method(self):
         """
         Setup method
         """
         self.log.info("Prerequisite: Deploy cortx cluster")
+        deploy_start_time = time.time()
+        self.log.info("Printing start time for deployment %s: ", deploy_start_time)
         self.log.info("Cleanup: Destroying the cluster ")
         resp = self.deploy_lc_obj.destroy_setup(self.csm_obj.master, self.csm_obj.worker_list,
```

Comment:
Do we need this ?

Reply:
we need this for destroying existing cluster and deploying fresh one right?